android/src/main/java/com/swmansion/enriched/EnrichedTextInputViewManager.kt

```diff
@@ -268,6 +272,22 @@ class EnrichedTextInputViewManager : SimpleViewManager<EnrichedTextInputView>(),
     view?.addMention(text, indicator, attributes)
   }
 
+  override fun requestHTML(view: EnrichedTextInputView?, requestId: Int) {
+    if (view == null) return
+
+    val spannable = view.text as? android.text.Spannable
```

Comment:
```suggestion
    val spannable = view.text as? Spannable
```

Also, remember to add missing (after such change) import